### PR TITLE
fix XCVM SPEC layout

### PR DESCRIPTION
**Pull request by dzmitry-lahoda**
Signed-off-by: Dzmitry Lahoda <dzmitry@lahoda.pro>

## Issue
- broken md

## Checklist
 - look at md layout is ok

### CODE REVIEW

File: code/xcvm/SPEC.md [hunk 1]
```diff
@@ -235,7 +235,7 @@ sequenceDiagram
 The call instruction supports bindings values on the executing side of the program by specifying the `Bindings`. This allows us to construct a program that uses data only available on the executing side. For example, the swap call of the following smart contract snippet expects a `to` address to receive the funds after a trade.
 
 ```rust
-fn swap(amount: u256, pair: (u128, u128), to: AccountId) { ... } 
+fn swap(amount: u256, pair: (u128, u128), to: AccountId) { .A.. }
```
Comment (aeryz) [inline code/xcvm/SPEC.md]: I think this 'A' is added accidentally

**Review by aeryz — CHANGES_REQUESTED** [review]

**Review by hussein-aitlahcen — APPROVED** [review]

**Comment by dzmitry-lahoda** [comment]
it has crazy failure
```
Download action repository 'actions/checkout@v3' (SHA:2541b1294d2704b0964813337f33b291d3f8596b)
Error: Can't use 'tar -xzf' extract archive file: /home/runner/_work/_actions/_temp_4a9f6896-9470-4d5f-84cf-6[24](https://github.com/ComposableFi/composable/actions/runs/3196350168/jobs/5218635604#step:1:25)cc7260a03/e5526187-dc86-4912-b017-de8ae29699e2.tar.gz. return code: 2.
```
retried 2 times already

**Comment by dzmitry-lahoda** [comment]
@Mergifyio refresh

**Comment by dzmitry-lahoda** [comment]
dozens of failures, but he just wanted only his simple 
```
fatal: unable to access 'https://github.com/composablefi/ics23/': Empty reply from server
error: program 'git' failed with exit code 128
````

**Comment by KaiserKarel** [comment]
@mergifyio refresh

**Comment by dzmitry-lahoda** [comment]
@Mergifyio refresh

**Pull request merged**